lib/StaticNarrative/exporter/app_processor.py

```diff
@@ -134,22 +110,20 @@ def _translate_param_value(
             valid_ws_types = param_spec.get("text_options", {}).get("valid_ws_types", [])
             if len(valid_ws_types) > 0 and value:
                 if isinstance(value, list):
-                    value = [upas[v][1] if v in upas else v for v in value]
+                    value = [
+                        self.config.indexed_data[v]["object_info"][1]
```

Comment:
magical set of kbase objects, indexed by ID, that already contain the data we're looking for